cmake/assets/run_nodered_template.sh.in

```diff
@@ -1,3 +1,17 @@
-cd @EVEREST_DOCKER_DIR@
-docker build -t everest-nodered .
-docker run --rm --network host --name everest_nodered --mount type=bind,source=@FLOW_FILE@,target=/data/flows.json everest-nodered
+# In the following a volume is created to contain the nodered config, this is done to
+# allow starting the nodered container from inside a devcontainer
+
+# Create docker volume to contain nodered config
+docker volume create nodered-config-volume
+
+# Create temporarily container to copy nodered config into the created volume
+docker create -v nodered-config-volume:/data --name nodered-config-container tianon/true:latest true
```

Comment:
does this really have to rely on tianon/true:latest? I would prefer to have something (probably slightly larger) but everest-hosted here

Reply:
We could choose any image here, tianon/true seems to be the smallest (fastest when downloading), options:

1. Rehost `tianon/true`, by basing an own image on it with no modifications
2. Use `debian:12-slim`, since the most other images are based on it, the chance having it on the system already is relative high

Reply:
I'm leaning towards the second option

Reply:
Updated